cypress/support/cypress-commands/commands.js

```diff
@@ -1263,3 +1263,90 @@ Cypress.Commands.add('methodOrEventResponseValidation', (validationType, request
     }
   });
 });
+
+/**
+ * @module commands
+ * @function validateMethodOrEventResponseForDynamicConfig
+ * @description Validating the event or method response
+ * @param {String} validationType - Determines event or method validation
+ * @param {String} method - API name
+ * @param {*} validationJsonPath - Contains an array or string with the path of the response value that needs to be validated.
+ * @param {String} contentObject - Contains source of truth for content validation.
+ * @param {String} appId - app identified name.
+ * @param {String} errorContent - Holds the error content validation object when error is expected.
+ * @param {String} eventExpected - Determines whether expecting for a event or not.
+ * @example
+ * cy.validateMethodOrEventResponseForDynamicConfig('method', 'account.id', 'result', {}, 'test.test', 'errorContent', 'eventExpected');
+ */
+Cypress.Commands.add(
+  'validateMethodOrEventResponseForDynamicConfig',
+  (
+    validationType,
+    method,
+    validationJsonPath,
+    contentObject,
+    appId,
+    errorContent,
+    eventExpected
+  ) => {
+    // Reading the appId from the environment variable
+    appId = UTILS.fetchAppIdentifierFromEnv(appId);
+    const context = {};
+    const expectingError = errorContent ? true : false;
+    contentObject = contentObject ? contentObject : CONSTANTS.NULL_RESPONSE;
```

Comment:
can we add comments for this if else condition

Reply:
added the comment